.github/workflows/ci.yml

```diff
@@ -55,16 +55,38 @@ jobs:
 
       - name: Doctest - PyVortex
         run: |
-          make clean && make doctest
+          make clean && uv run make doctest
         working-directory: docs/
 
       - name: Ensure docs build - PyVortex
+        run: |
+          uv run make -C docs html
+
+      - name: Ensure wheel and sdist can be built on Linux - PyVortex
+        uses: PyO3/maturin-action@v1
+        env:
+          # false, off, and 0 all work: https://github.com/mozilla/sccache/blob/main/src/config.rs#L620
+          SCCACHE_GHA_ENABLED: false
+        with:
+          rust-toolchain: ${{ steps.rust-toolchain.version }}
+          working-directory: pyvortex
+          target: x86_64
+          manylinux: auto
+          args: --release --interpreter python3.10 --sdist
+
+      - name: Ensure wheel can run tests successfully
         run: |
           set -ex
 
-          (cd pyvortex && uv run maturin develop)
+          echo creating a fresh venv to ensure wheel dependencies are correct
+
+          python3 -m venv .venv2
```

Comment:
I think we should use uv's ability to create a new virtual env here, so the version is as expected instead of whatever the runner is installed on the runner.

Reply:
You mean `uv venv .venv2`, yeah?

Reply:
yup

Reply:
Done.